Lib/test/test_types.py

```diff
@@ -642,6 +642,11 @@ def test_traceback_and_frame_types(self):
     def test_capsule_type(self):
         self.assertIsInstance(_datetime.datetime_CAPI, types.CapsuleType)
 
+    def test_timezone_base_type(self):
+        class MyTimezone(_datetime.timezone):
+            pass
+
+        self.assertIs(issubclass(MyTimezone, _datetime.timezone), True)
```

Comment:
Test that `MyTimezone(...)` returns a `MyTimezone` instance.

Reply:
It's not  :disappointed: Thanks for finding this :dart: